examples/tutorial 1a - Using a Quantum Device to solve MIS.ipynb

```diff
@@ -107,7 +176,7 @@
    "cell_type": "markdown",
```

Comment:
We should do something with the data, at the very least print it and comment on it a bit.